src/Adapter/Order/AbstractOrderHandler.php

```diff
@@ -30,32 +30,25 @@
 use Cart;
 use Combination;
 use Configuration;
-use Context;
 use Currency;
 use Customer;
 use Group;
 use Order;
-use PrestaShop\Decimal\Number;
 use PrestaShop\PrestaShop\Adapter\Number\RoundModeConverter;
 use PrestaShop\PrestaShop\Core\Domain\Order\Exception\OrderException;
 use PrestaShop\PrestaShop\Core\Domain\Order\Exception\OrderNotFoundException;
 use PrestaShop\PrestaShop\Core\Domain\Order\ValueObject\OrderId;
 use PrestaShop\PrestaShop\Core\Domain\Product\ValueObject\ProductId;
 use PrestaShop\PrestaShop\Core\Localization\CLDR\ComputingPrecision;
-use PrestaShopDatabaseException;
 use PrestaShopException;
 use Product;
-use SpecificPrice;
-use TaxManagerFactory;
 use Validate;
 
 /**
  * Reusable methods for Order subdomain command/query handlers.
  */
 abstract class AbstractOrderHandler
 {
-    const COMPARISON_PRECISION = 6;
```

Comment:
BC Break to remove a public constant

Reply:
It's not public as it was only used in this abstract class, it's just that we didn't qualify it as private
But it can be considered as a BC I can flag the PR as BC if you prefer, but anyway I think it's acceptable

Or I can keep this now useless const to avoid BC? 🤷‍♂️